rules/capacityfactors.smk

```diff
@@ -52,7 +52,7 @@ rule time_average_capacityfactor_map:
     input:
         scripts = script_dir + "capacityfactors/averages_map.py",
         id_map = "build/capacityfactors/{technology}-ids.tif",
-        timeseries = rules.capacityfactor_timeseries.output
+        timeseries = "build/capacityfactors/{technology}-timeseries.nc"
```

Comment:
Why did you move away from the `rule` mechanism here? I'd say we should always use it when possible.

Reply:
When using wildcards in rule outputs, does the wildcard propagate through to the rule being called? If so, I'll revert this and a couple of others to using the rule mechanism.

Reply:
I think so, yes. Otherwise this couldn't have worked before, no?

Reply:
fair.